src/components/views/context_menus/MessageContextMenu.js

```diff
@@ -253,6 +253,16 @@ module.exports = createReactClass({
         this.closeMenu();
     },
 
+    onCopyClick: function(e: Event) {
+        const textField = document.createElement('textarea');
+        textField.innerText = this.props.mxEvent.event.content.body;
```

Comment:
event body does not have to match what is rendered due to the html portion of events so this could easily be abused to make people copy-paste things they didn't know they were.

Reply:
Yet clients do, EW/ED will use `body` for desktop notifications for example.